### DISCUSSION TITLE
monitor interval

### DISCUSSION THREAD

**Discussion by ladnyik** [abraunegg/onedrive#2899] [Q&A] [answered]
Hi
What is the minimum of monitor_interval ? I tried to set it e.g. 60 it says invalid key value.

**Comment by abraunegg** [comment, marked as answer]
Please read the documentation:
* https://github.com/abraunegg/onedrive/blob/master/docs/usage.md
* https://github.com/abraunegg/onedrive/blob/master/docs/application-config-options.md
* https://github.com/abraunegg/onedrive/blob/master/docs/client-architecture.md
